test/conformance/ptp/ptp.go

```diff
@@ -866,7 +929,7 @@ func getNonPtpMasterSlaveAttachedInterfaces(pod *v1core.Pod) []string {
 }
 
 func enablePTPEvent() error {
-	ptpConfig, err := client.Client.PtpV1Interface.PtpOperatorConfigs(PtpLinuxDaemonNamespace).Get(context.Background(), "default", metav1.GetOptions{})
+	ptpConfig, err := client.Client.PtpV1Interface.PtpOperatorConfigs(PtpLinuxDaemonNamespace).Get(context.Background(), PtpLinuxDaemonNamespace, metav1.GetOptions{})
```

Comment:
Is this in "openshift-ptp" or "default"?

Reply:
good catch, it shall be default. Thanks!